R/ppc-intervals.R

```diff
@@ -319,9 +325,11 @@ label_x <- function(x) {
            size = 1,
            grouped = FALSE,
            style = c("intervals", "ribbon"),
-           x_lab = NULL) {
+           x_lab = NULL,
+           y_draw = c("line", "points", "both")) {
 
   style <- match.arg(style)
+  y_draw <- match.arg(y_draw)
```

Comment:
The `match.arg` function will result in an error if the user's `y_draw` argument isn't one of the three valid ones.